source/MRMesh/MRIOParsing.h

```diff
@@ -20,10 +20,18 @@ template<typename T>
 VoidOrErrStr parseObjCoordinate( const std::string_view& str, Vector3<T>& v, Vector3<T>* c = nullptr );
 template<typename T>
 VoidOrErrStr parsePtsCoordinate( const std::string_view& str, Vector3<T>& v, Color& c );
+
+// reads the number of polygon points
+VoidOrErrStr parseNumPoint( const std::string_view& str, int* numPoints );
```

Comment:
better to write comment like reads first integer from the line (it can be anything), mb change name too